doc/api/util.md

```diff
@@ -2024,6 +2024,23 @@ util.types.isAnyArrayBuffer(new ArrayBuffer());  // Returns true
 util.types.isAnyArrayBuffer(new SharedArrayBuffer());  // Returns true
 ```
 
+### `util.types.isArrayBufferDetached(value)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `value` {any}
+* Returns: {boolean}
+
+Returns `true` if the value is a built-in [`ArrayBuffer`][] and
+is detached.
```

Comment:
A comment here quickly explaining what "detached" means might be helpful to newcomers.

Reply:
Thanks for the review. I updated the pull request.